src/applications/disability-benefits/all-claims/tests/all-claims.cypress.helpers.js

```diff
@@ -0,0 +1,176 @@
+import moment from 'moment';
+
+import { VA_FORM_IDS } from 'platform/forms/constants';
+
+/* eslint-disable camelcase */
+export const mockUser = {
```

Comment:
It would probably be cleaner to create fixtures of these mock responses and give them a common place to live, but just wanted to get the tests working for now.

Another idea to consider when we introduce Pact is creating a helper that will pull the appropriate responses from generated pact files using one of the interaction attributes as the key (`state`? `uponReceiving`?).